packages/dhis2/README.md

```diff
@@ -14,6 +14,12 @@ core execute \
  -o ./tmp/output.json
 ```
 
+> **Important Note:**  
+> This adaptor uses the old tracker version. For the new version, use 6.0. The
```

Comment:
Can we say "use adaptor 6.0+"

Also, any API version less than 42 WILL work :sweat_smile: Project confidence :muscle: